v1.0 fails to start after update

### Component

server

### Describe the bug

After updating to the new v1.0 Docker image Woodpecker fails to start with the following error message. Downgrading to v1.0.0 fixes it.

```
panic: handlers are already registered for path '/metrics'

goroutine 1 [running]:
github.com/gin-gonic/gin.(*node).addRoute(0x13d0062?, {0xc00046bf28, 0x8}, {0xc00049ede0?, 0xc, 0xc})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/gin-gonic/gin/tree.go:260 +0x734
github.com/gin-gonic/gin.(*Engine).addRoute(0xc0004029c0, {0x13d0062, 0x3}, {0xc00046bf28, 0x8}, {0xc00049ede0, 0xc, 0xc})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/gin-gonic/gin/gin.go:335 +0x272
github.com/gin-gonic/gin.(*RouterGroup).handle(0xc0004029c0, {0x13d0062, 0x3}, {0x13dfd43?, 0x0?}, {0xc00053bf00, 0x1, 0x0?})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/gin-gonic/gin/routergroup.go:89 +0x145
github.com/gin-gonic/gin.(*RouterGroup).GET(...)
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/gin-gonic/gin/routergroup.go:117
github.com/woodpecker-ci/woodpecker/server/router.Load(0x13fb9b8?, {0xc00052d8f0, 0x4, 0x4})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/server/router/router.go:74 +0x685
main.run(0xc0003fc640)
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/cmd/server/server.go:201 +0xd70
github.com/urfave/cli/v2.(*Command).Run(0xc000022580, 0xc0003fc640, {0xc0001181e0, 0x1, 0x1})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/urfave/cli/v2/command.go:274 +0x9eb
github.com/urfave/cli/v2.(*App).RunContext(0xc0006de000, {0x16afed8?, 0xc000124000}, {0xc0001181e0, 0x1, 0x1})
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/urfave/cli/v2/app.go:332 +0x616
github.com/urfave/cli/v2.(*App).Run(...)
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/vendor/github.com/urfave/cli/v2/app.go:309
main.main()
        /woodpecker/src/github.com/woodpecker-ci/woodpecker/cmd/server/main.go:38 +0x254
```

### System Info

```shell
Woodpecker v1.0 (Docker 24.0.5 and Forgejo 1.20)
```


### Additional context

_No response_

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

--- a/server/router/router.go
+++ b/server/router/router.go
@@ -71,10 +71,6 @@ func Load(noRouteHandler http.HandlerFunc, middleware ...gin.HandlerFunc) http.H
 		base.GET("/healthz", api.Health)
 	}
 
-	e.GET("/metrics", metrics.PromHandler())
-	e.GET("/version", api.Version)
-	e.GET("/healthz", api.Health)
-
 	apiRoutes(base)
 	setupSwaggerConfigAndRoutes(e)
 

